packages/node/test/integrations/undici.test.ts

```diff
@@ -51,10 +51,10 @@ conditionalTest({ min: 16 })('Undici integration', () => {
   it.each([
     [
       'simple url',
-      'http://localhost:18099',
+      'http://localhost:18100',
```

Comment:
Yes - this was because `18099` was used by another test so running the tests in parallel would lead to issues. See: https://github.com/getsentry/sentry-javascript/pull/8449#issuecomment-1629467068